src/Footer/ActiveCustomer/ActiveCustomer.jsx

```diff
@@ -1,14 +1,14 @@
-
-
-function ActiveCustomer() {
+import "./ActiveCustomer.css";
+export const ActiveCustomer = ({ arrayToRender }) => {
+  const activeCustomes = arrayToRender.filter(x => { if (x.status === "active") { return x.status } })
   return (
     <div className="left-footer">
-                <span>active customers: 
-                    <span id="active-customers" style="font-weight: bold;"></span>
-                <span id="length"></span>
-                </span>
-            </div>
-      
+      <span>active customers:
+        <i id="active-customers" style={{ fontWeight: "bold" }}>{activeCustomes.length}</i>
```

Comment:
Don't style in JSX, use classes
```suggestion
        <i id="active-customers" className="...">{activeCustomes.length}</i>
```

Reply:
ok